docs/developing/gasless.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Gasless transactions
+proofedDate: na
+iterationBy: na
+includedInSite: true
+approvedBy: na
+comment: There are code examples (React>> https://codesandbox.io/s/t3fid3) and https://thirdweb.com/neon-evm-devnet/0xF3eBc32292F4BbFB83DECB97Eb42d95da968f775/sources >> bit confused though, the flow described means that the dApp dev has nothing to do but accept the 0 cost tx. If these code examples "demonstrate how to implement gasless transactions" does that mean they apply to Proxy Operators?
```

Comment:
@eubash I have started a page for the dApp dev to understand gasless
I am guessing that the code examples must be for Proxy Operators -- in which case, we need another page. PTAL and let me know!

Reply:
@m4sterbunny, not really. The code examples are for dApps, we don't need some example page for proxy operators, maybe just a piece of configuration. I'll get back to you after I clarify this with @YYYurchenko.